mobile/src/main/res/values/strings.xml

```diff
@@ -121,5 +121,16 @@
     <string name="notification_posted">Notification posted</string>
     <string name="pref_default_maximum_retries">1</string>
     <string name="pref_title_maximum_retries">Maximum Retries</string>
+    <string name="action_about">About</string>
+    <string name="action_home_mycroft_ai">Home.Mycroft.Ai</string>
```

Comment:
The action name doesn't fully convey that this is the mycroft website - how about `Check out our website` or `Mycroft Homepage` or something like that?